CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Unreleased
+
+### Dependencies
+
+- Bump the version of the .NET SDK that we use from 9.0.203 to 9.0.300 ([#4259](https://github.com/getsentry/sentry-dotnet/pull/4259))
+  - Note that this also required we bump various Java depdendencies (since version 9.0.300 of the Android workload requires newer versions of the these)
```

Comment:
```suggestion
  - Note that this also required we bump various Java dependencies (since version 9.0.300 of the Android workload requires newer versions of the these)
```